### PR TITLE
V6 Theme Fixes

### LINKED ISSUE #47: Transition all used a lot
**Enhancement**

We should not use `transition: all` in our css as it can be blocking for the browser.

### PR DESCRIPTION
**Pull request by tomdye**
**Type:** bug

The following has been addressed in the PR:

* [ ] There is a related issue
* [ ] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [ ] Unit or Functional tests are included in the PR

**Description:**
- styles helper text
- styles leading / trailing sections of text-input
- removes use of `transition: all`
- fixes min-width on select native element
- runs `tcm` to generate correct css typings

Resolves #47 #40 #39 #38 

Screenshot 2019-03-25 20 42 49

### CODE REVIEW

File: src/dojo/index.ts [hunk 1]
```diff
@@ -4,14 +4,15 @@ import * as calendar from './calendar.m.css';
 import * as checkbox from './checkbox.m.css';
```
Comment (tomdye) [inline src/dojo/index.ts:4]: these have been alphabetised

File: src/dojo/combobox.m.css [hunk 2]
```diff
@@ -54,7 +54,6 @@
 	box-shadow: var(--box-shadow-dimensions-small) var(--color-box-shadow);
 	display: none;
 	position: absolute;
-	transition: all var(--transition-duration) var(--transition-easing);
```
Comment (tomdye) [inline src/dojo/combobox.m.css:57]: this class is not longer being transitioned

File: src/dojo/text-input.m.css [hunk 3]
```diff
@@ -1,37 +1,45 @@
 @import './variables.css';
 
-.root, .root * { box-sizing: border-box; }
```
Comment (tomdye) [inline src/dojo/text-input.m.css:3]: `*`  selectors are no longer to be used, they break css-module rules

**Review by rorticus — APPROVED** [review]
This all looks sensible to me! 👍

**Pull request merged**